connector_search_engine/models/se_index.py

```diff
@@ -372,11 +372,14 @@ def resynchronize_all_bindings(self):
             item_ids = []
             adapter = self.se_adapter
             binding_model = self.env[index.model_id.model]
-            for index_record in adapter.each():
-                ext_id = adapter._get_odoo_id_from_index_data(index_record)
-                binding = binding_model.browse(ext_id).exists()
-                if not binding:
-                    item_ids.append(ext_id)
+            for index_record in adapter.each(fetch_fields=["id"]):
```

Comment:
@sebastienbeau If you modify the method signature, youmust modify the related ABC class https://github.com/OCA/search-engine/blob/16.0/connector_search_engine/tools/adapter.py#L41

Reply:
Thanks for the catch !!!
I fix it : 
https://github.com/OCA/search-engine/pull/211/files#diff-0999632348627c0ffd09fd3aa1bbe05571cca744693515a5a7d28e92d12cb722R41